src/search/search.cpp

```diff
@@ -971,8 +927,10 @@ search_req _HandleSearchRequest(CTCPRequestPacket& PTCPRequest)
     }
 
     return sr;
-    // не обрабатываем последние биты, что мешает в одну кучу например "/blacklist delete Name" и "/sea all Name"
+    // не обрабатываем последние биты, что мешает в одну кучу
```

Comment:
This was the only comment in these files that I couldn't quite understand enough to translate.

It's something about skipping the final bits, but by reading through the code it's not obvious why this is done or what's happening.

The phrase `кучу` here could possibly mean "heap", "struct", or "packet". Or perhaps something else...

Reply:
They might be referring to extra information that's usually supposed to be in the same packet, but is being skipped because it's not always there.

I've probably booted the search server _once_, and I'm not finding anything useful in Windower's packet library, so I couldn't guess what that information might potentially be.

Reply:
What's your preference for this one? I was tempted to just leave it as is.

I could drop in a more literal translation of the text, but I worry if it's not accurate then it won't be meaningful.